src/main/java/com/spartronics4915/frc2020/commands/SetBlingStateCommand.java

```diff
@@ -0,0 +1,48 @@
+package com.spartronics4915.frc2020.commands;
```

Comment:
I would recommend  structuring the Bling subsystem to be consistent with the rest of the codebase by creating a `BlingCommands` file and having individual subclasses (each calling methods) for each Bling state similar to what's being done in #42 .

If you think that conversion will be too confusing to involved parties, don't bother - states still work with the Command framework.

Reply:
I don't see any benefit from creating a method for each type of animation in robot code. In addition, I see it as a maintainability issue as we need to make sure each supported animations are matched on the Arduino codebase. Having a list of enums on both side of the system, i.e. robot code and Arduino, simplifies integration.

Reply:
Ideally, we'd be able to call the actual C++ methods (`setPixel`, `showPixels`, etc) in the Command itself, but I doubt that's possible (or good)....

I would still recommend having multiple Commands within a class, but each of them calling `mLED.setBlingState`.

Reply:
@j-james Robot code does NOT do any pixel manipulation. It is all done in the Arduino. Robot code only sends the intent using enum values (such as BlingState.INTAKE_DOWN) that are mapped to specific animation methods in the Arduino. These animations also listed as enums, which match to robot code for easy maintenance. 

I do not agree with creating multiple methods that just maps back to BlingState enums. That will be a maintenance headache. However, there is nothing stopping you from taking this on. I am not planning on making any more changes to the code.